CHANGELOG.md

```diff
@@ -1,71 +1,23 @@
 # Changelog
 
-All notable changes to Docker and Docker Compose resources for WSO2 Identity and Management version `5.11.x`
+All notable changes to Docker and Docker Compose resources for WSO2 Identity and Management version `6.0.x`
 in each resource release, will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
-## [v5.11.0.15] - 2022-06-20
+## v6.0.0.1 - 2022-08-10
 
 ### Changed
-- Update alpine base docker image version to 3.16
```

Comment:
Why we removed all the changelog is that a usual practice?

Reply:
Lets not remove the change log in master. We need to create a separate branch for 6.0.0. There we can remove the change log content

Reply:
It is for the new release, we remove all the existing logs and adding 6.0.0 ones only. And will branch out 6.0.x from the master.

Reply:
Ack. Will adhere.

Reply:
But the changleog is specifc to 5.11 as per 
```
Changelog
All notable changes to Docker and Docker Compose resources for WSO2 Identity and Management version 5.11.x in each resource release, will be documented in this file.

The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
```
So, we anyways need to update this master branch one right?

Reply:
And please refer 5.11 https://github.com/wso2/docker-is/pull/265/files